### PR TITLE
ElasticSearch: Add keywords to plugin.json

### PR DESCRIPTION
**Pull request by nmarrs**
Update plugin.json keywords to include "datasource" + "database"

Related to https://github.com/grafana/oss-plugin-partnerships/issues/1055

### CODE REVIEW

File: public/app/plugins/datasource/elasticsearch/plugin.json [hunk 1]
```diff
@@ -3,27 +3,29 @@
...
   "category": "logging",
-
   "info": {
     "description": "Open source logging & analytics database",
     "author": {
       "name": "Grafana Labs",
       "url": "https://grafana.com"
     },
-    "keywords": ["elasticsearch"],
+    "keywords": ["elasticsearch", "datasource", "database"],
```
Comment (iwysiu) [inline public/app/plugins/datasource/elasticsearch/plugin.json:12]: should we also add the "logs" or "metrics" keywords here?

Comment (nmarrs) [reply public/app/plugins/datasource/elasticsearch/plugin.json]: good point, added in latest commit 👍

File: public/app/plugins/datasource/elasticsearch/plugin.json [hunk 2]
```diff
@@ -3,27 +3,29 @@
...
   "category": "logging",
-
   "info": {
     "description": "Open source logging & analytics database",
     "author": {
       "name": "Grafana Labs",
       "url": "https://grafana.com"
     },
-    "keywords": ["elasticsearch"],
+    "keywords": ["elasticsearch", "datasource", "database", "logs", "metrics"],
```
Comment (iwysiu) [inline public/app/plugins/datasource/elasticsearch/plugin.json:12]: I realize i wasn't very clear, but i listed logs and metrics as examples, based on opensearch i suspect we should be adding these instead:   https://github.com/grafana/opensearch-datasource/blob/a66f595e6be35a3b131b234ffd32b998152bcb1f/src/plugin.json#L19
```suggestion
    "keywords": ["elasticsearch", "datasource", "database", "logs", "nosql", "traces"],
```

**Review by iwysiu — APPROVED** [review]
approved with one change

File: public/app/plugins/datasource/elasticsearch/plugin.json (hunk 2 above)
Comment (nmarrs) [reply public/app/plugins/datasource/elasticsearch/plugin.json]: ah whoops, makes sense. Updated in latest commit

**Pull request merged**